views/install.sh.erb

```diff
@@ -439,8 +447,13 @@ install_file() {
   echo "Installing $project $version"
   case "$1" in
     "rpm")
-      echo "installing with rpm..."
-      rpm -Uvh --oldpackage --replacepkgs "$2"
+      if test "x$platform" = "xnexus" || test "x$platform" = "xios_xr"; then
+        echo "installing with yum..."
+        yum install -yv "$2"
```

Comment:
I wish it was incorrect. :)  On the nexus and ios_xr platforms, `rpm` installs to volatile storage which is wiped on reboot.  You must use `yum` to have it persist.